### DISCUSSION TITLE
Adding multiple entries to the menu?

### DISCUSSION THREAD

**Discussion by gerroon** [buggyj/tiddlyclip-plugin#115] [Q&A] [answered]
Hi

I would like to have additional entries int he right clicik menu with

snip.txt
snip.markdown
snip.html

Basically I want to be able to have multiple versions of snipping that can capture at different formats.

thanks

**Comment by buggyj** [comment, marked as answer]
Only html and  text are supported.

You can include a newline in the table in the tiddler tiddlyclipconfig like so

![image](https://user-images.githubusercontent.com/2855795/220429078-5c109c24-ab07-4295-a566-816dc4091fde.png)

**Reply by arunnbabu81** [reply]
https://github.com/buggyj/tiddlyclip-plugin/discussions/79#discussioncomment-2299068

You can modify the rule given in this post for the markdown content. I don't use markdown format in my wiki. So I don't have much experience with it
